docs/TopK_Commands.md

```diff
@@ -184,12 +184,13 @@ TOPK.COUNT test foo 42 nonexist
 Return full list of items in Top K list.
 
 ```
-TOPK.LIST {key}
+TOPK.LIST {key} [WITHCOUNT]
 ```
 
 ### Parameters
 
 * **key**: Name of sketch where item is counted.
+* **WITHCOUNT**: Count of each element is returned.  
 
 ### Complexity
 
```

Comment:
@ashtul we're missing the changes to the reply. 
Also, we need to propagate the awareness of this change into all clients.

Reply:
We don't always put output for all optional params as it makes the docs very long.